FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -1042,23 +1069,55 @@ float Maslow_::computeTL(float x, float y, float z){
     return sqrt(a*a+b*b+c*c) - (_beltEndExtension+_armLength);
 }
 
-void Maslow_::setTargets(float xTarget, float yTarget, float zTarget){
+/*
+* This computes the target lengths of the belts based on the target x and y coordinates
+* and sends that information to each arm.
+*/
+void Maslow_::setTargets(float xTarget, float yTarget, float zTarget, bool tl, bool tr, bool bl, bool br){
 
-    //Scaling to correct size
-    //xTarget = xTarget;
-    //yTarget = yTarget;
-    
-//    if(!calibrationInProgress){
+        //Store the target x and y coordinates for the getTargetN() functions
+        targetX = xTarget;
+        targetY = yTarget;
+        targetZ = zTarget;
 
         computeTensions(xTarget, yTarget);
 
-        axisBL.setTarget(computeBL(xTarget, yTarget, zTarget));
-        axisBR.setTarget(computeBR(xTarget, yTarget, zTarget));
-        axisTR.setTarget(computeTR(xTarget, yTarget, zTarget));
-        axisTL.setTarget(computeTL(xTarget, yTarget, zTarget));
-//    }
+        if(tl){
+            axisTL.setTarget(computeTL(xTarget, yTarget, zTarget));
+        }
+        if(tr){
+            axisTR.setTarget(computeTR(xTarget, yTarget, zTarget));
+        }
+        if(bl){
+            axisBL.setTarget(computeBL(xTarget, yTarget, zTarget));
+        }
+        if(br){
+            axisBR.setTarget(computeBR(xTarget, yTarget, zTarget));
+        }
 }
 
+/*
+* Get's the most recently set target position in X
+*/
+double Maslow_::getTargetX(){
```

Comment:
I added these getTarget() functions. I'm not 100% sold that I love the way I did this, but I think that any time there is a set() function there should be a get() function too. 

Probably the most correct way to do this would be to read the position from each arm and use the belt lengths to compute the XY coordinates...but the problem with that is that we want to use this during the calibration process when we don't actually have good coordinates for the frame yet